dubbo-rpc/dubbo-rpc-thrift/src/main/java/org/apache/dubbo/rpc/protocol/thrift/ThriftProtocol.java

```diff
@@ -63,9 +63,9 @@
 
             if (msg instanceof Invocation) {
                 Invocation inv = (Invocation) msg;
-                String serviceName = inv.getAttachments().get(Constants.INTERFACE_KEY);
+                String path = inv.getAttachments().get(Constants.PATH_KEY);
```

Comment:
Should we adjust the implementation of this method：
`org.apache.dubbo.remoting.exchange.support.ExchangeHandlerAdapter#reply`
so that we can use PATH to get the corresponding service?

It seems that INTERFACE is still being used to get services.

Reply:
We use `exporterMap.get(serviceKey);` to get the exporter.
And `serviceKey` is consist of: `group/path:version:port`

Reply:
I don't understand why we are still using INTERFACE.

Reply:
My bad...
Nothing..